docs/application/native/guides/internationalization/i18n.md

```diff
@@ -783,17 +783,17 @@ The UCharIter functions return code unit values in the range of 0 - 0xffff. Befo
 
 The PluralFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) applications) allows you to [format defined plural words](#plural_format_examples). Alongside with PluralRules module, you can define rules to which format would apply.
 
-Plural Format supports the creation of internationalized messages with plural inflection.
-It is based on plural selection, i.e. the caller specifies messages for each plural case
-that can appear in the user's language and the Plural Format selects the appropriate message
-based on the number.
+Plural Format supports creating locale-dependent messages with plural inflection.
+To enable plural formatting, user have to specify each message for every plural case
+that will be used, based on the desired language. Then the Plural Format module will be
+selecting appropriate text based on provided number.
 
 ### The Problem of Plural Forms in Internationalized Messages:
 
-Different languages have different ways to inflect plurals. Creating internationalized messages
-that include plural forms is only feasible when the framework is able to handle plural forms
-of all languages correctly. In some languages, like Polish, one plural case applies to infinitely many intervals
-(e.g., the plural case applies to numbers ending with 2, 3, or 4 except those ending with 12, 13, or 14).
+Many languages have different approach to create plural nouns. To create locale dependent messages is only
+workable when framework enables to manage plural forms of all languages correctly. For example in Polish,
+one plural case applies to infinitely many intervals (e.g., the plural case applies to numbers ending with
+2, 3, or 4 except those ending with 12, 13, or 14).
```

Comment:
Please change as following:

Many languages have different approach to create plural nouns. The locale-dependent message is workable when the framework enables to manage plural forms of all language correctly.
For example, in Polish, one plural case applies to infinity many intervals (For example, the plural case applies to numbers ending with 2, 3, or 4 except those ending with 12, 13, or 14).

Reply:
Changes also last sentence of this paragraph to be described as it is in Polish language